clang/lib/Format/WhitespaceManager.cpp

```diff
@@ -471,6 +471,9 @@ AlignTokenSequence(const FormatStyle &Style, unsigned Start, unsigned End,
            Previous >= 0 &&
            Changes[Previous].Tok->getType() == TT_PointerOrReference;
            --Previous) {
+        // Don't align function default argument using return type maximum size
+        if (Changes[Previous + 1].Tok->is(tok::equal))
+          continue;
```

Comment:
I have multiple issues with this snippet:
* I don't understand the comment.
* Comments end in full stop.
* You execute this in a loop, but it is either true on the first run or never.

Reply:
I pushed a new patch, to move the check out of the loop